swap/src/bin/asb.rs

```diff
@@ -26,12 +26,13 @@ use swap::asb::command::{parse_args, Arguments, Command};
 use swap::asb::config::{
     initial_setup, query_user_for_initial_config, read_config, Config, ConfigNotInitialized,
 };
+use swap::asb::event_loop::{EventLoop, KrakenRate};
+use swap::asb::recovery::redeem;
```

Comment:
You are importing from the submodule even though we re-export. Making these modules private should allow us to be more consistent here.